cmd/kubemark/hollow-node.go

```diff
@@ -93,18 +93,20 @@ func (c *hollowNodeConfig) addFlags(fs *pflag.FlagSet) {
 }
 
 func (c *hollowNodeConfig) createClientConfigFromFile() (*restclient.Config, error) {
-	clientConfig, err := clientcmd.LoadFromFile(c.KubeconfigPath)
+	clientConfigs, err := clientcmd.LoadFromFile(c.KubeconfigPath)
```

Comment:
I have made changes to kubeconfigfilepath to have multiple kubeconfig files. For example, /var/run/kubernetes/apiserver1.kubeconfig, /var/run/kubernetes/apiserver2.kubeconfig.

Just for review, no need to resolve.